droid-hal-device.inc

```diff
@@ -191,6 +197,29 @@ echo Running droid build in HABUILD_SDK
 ubu-chroot -r /srv/mer/sdks/ubu "cd %android_root; source build/envsetup.sh; lunch %{device}%{?device_variant}; rm -f .repo/local_manifests/roomservice.xml; make %{?_smp_mflags} %{?hadk_make_target}%{!?hadk_make_target:hybris-hal}"
 %endif
 
+# tools package: look first for device specific mk file, then use dhd's
+rm -f %{android_root}/system/core/mkbootimg/mkbootimg \
+      %{android_root}/system/core/libsparse/img2simg \
+      %{android_root}/system/core/libsparse/simg2img
+
+if test -f rpm/helpers/mkbootimg.mk; then
+    make -f $(pwd)/rpm/helpers/mkbootimg.mk -C %{android_root}/system/core/mkbootimg/
```

Comment:
We should keep it consistent with current "make" approach used here: https://github.com/mer-hybris/droid-hal-device/blob/f6bf842884bd5dfa458dd8b2b868337fc2c78d6f/droid-hal-device.inc#L205
It also needs testing whether this line (make -f $(pwd)/rpm... , or after fixing the bit above: (cd rpm/; make ...) ) will build on OBS wrt to correct paths. @lbt, I think your glance is enough, instead of full test build.